test/integration/targets/ec2_eip/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/aws
+unsupported
```

Comment:
Why is this test marked unsupported?

Reply:
The test needs either an `unsupported` or `shippable/aws/group*` alias. Using `unsupported` indicates the test cannot be run on our CI system. Use of a script based test (`runme.sh`) or role based test does not matter.

If the tests don't pass in CI, and thus require `unsupported`, can you explain why?

Reply:
correct. [here](https://app.shippable.com/github/ansible/ansible/runs/100887/69/tests) you can see a test tun with `shippable/aws/group2` as you can see it fails when calling `AssociateAddress operation`. I think due [this](http://docs.aws.amazon.com/AWSEC2/latest/APIReference/ec2-api-permissions.html#ec2-api-unsupported-resource-permissions) as stated [here](https://github.com/ansible/ansible/blob/abdcf2a776f4c382f6508766f9997fcf7e531764/hacking/aws_config/testing_policies/compute-policy.json#L33)

Reply:
It looks like we'll need to update the permissions in our CI system to allow these tests to run. I've applied the `needs_ci_update` label to the PR to track this.